utils/prod/update_config.py

```diff
@@ -83,33 +105,39 @@ def update_configs(mode: str = 'prod') -> None:
             print(f'Config file not found for {kit_name}. Skipping.')
             continue
 
-        with open(config_path, 'r') as file:
+        with open(config_path, 'r', encoding='utf-8') as file:
             config: Dict[str, Any] = yaml.safe_load(file)
 
         if mode == 'prod':
             update_config_prod(config)
         else:
             update_embedding_model(config)
 
-        with open(config_path, 'w') as file:
-            yaml.dump(config, file)
+        with open(config_path, 'w', encoding='utf-8') as file:
+            yaml.dump(config, file, default_flow_style=False)
```

Comment:
We need comments to explain why we don't want the default style.

Reply:
Setting default_flow_style=False ensures that the YAML is written in block style, which is more readable. I'll add a comment to explain this choice.